docs/source/testing/trip_test.rst

```diff
@@ -0,0 +1,99 @@
+
+* **How long do the trip tests take to run?** The code is re-compiled several times during the trip tests, so the majority of time will be spent compiling. Each test case is only run for a few days, so once the code has compiled it shouldn't take more than 30 mins or so, depending on how many cores you're using. You can increase the number of cores by using the ``-n`` option.
+
+* **I am getting an error message saying that git cannot find my commit ID - how do I fix this?** If you are specifying commit IDs that are present on your fork but not in the ExeClim version of Isca, you will need to specify your own fork as the repository. To do this, use the ``-r`` option, e.g. ``-r git@github.com:sit23/Isca``. Note that you can specify the repository location using the HTTPS option, or the SSH option depending on your preference.
+
+* **Do I have to connect to GitHub to run the trip tests, or can I do it all locally?** It is possible to run the trip tests locally. Use the option ``-r PATH_TO_MY_LOCAL_REPO`` and it should work fine. 
+
+* **Which of the test cases does the trip tests run**? A list of the available test cases can be found in the ``list_all_test_cases_implemented_in_trip_test`` function in ``trip_test_functions.py``.
+
+* **I haven't setup Socrates - does that matter?** No - just post the results you can obtain.
+
+* **If I add a new test case to the** ``test_cases`` **folder, how do I add it to the trip tests?** In the ``get_nml_diag`` function in ``trip_test_functions.py``, you will see several similar calls to import the namelist dictionary from each of the existing test cases. Just copy this syntax and edit it for your test case. You can then add the name of your test case to the list in the ``list_all_test_cases_implemented_in_trip_test`` function, and it should be run by default.
+
+* **I have defined a new** ``codebase`` **object for my test case. How can I get the trip test to select it?** In the ``conduct_comparison_on_test_case`` the codebase object is selected. You can edit this section accordingly. 
+
+* **Why don't you compare the results of the test cases to some standard output, rather than comparing the results from two commits?** We do this because different compilers and different hardware can produce slightly different results for a given test case. Therefore comparing to standard output is a much more difficult test for users to pass. We therefore compare the results of two different Isca versions *on the same hardware and with the same compilers* to make the tests easier to understand.
+
+Authors
+----------
+This documentation was written by Stephen Thomson, peer reviewed by X, and quality controlled by Y.
```

Comment:
can you add me for the review and ross for the QC please?

Reply:
Done